xrft/xrft.py

```diff
@@ -621,9 +636,18 @@ def power_spectrum(da, dim=None, scaling="density", **kwargs):
     dim : str or sequence of str, optional
         The dimensions along which to take the transformation. If `None`, all
         dimensions will be transformed.
+    real : str, optional
```

Comment:
```suggestion
    real_dim : str, optional
```

I think it would be easier to understand this argument if it were explicitly named `real_dim`.